elvee.c

```diff
@@ -455,3 +480,74 @@ void timestamp()
     month = (strstr(month_names, month_name) - month_names) / 4 + 1;
     printf("%04hu-%02hu-%02hu %02hu:%02hu:%02hu\n", year, month, day, hour, min, sec);
 }
+
+// Adapted from https://docs.microsoft.com/en-us/archive/blogs/twistylittlepassagesallalike/everyone-quotes-command-line-arguments-the-wrong-way
+// See issue #1
+// If quoting is needed then a newly allocated string is returned with with
```

Comment:
"with with"

Reply:
Thanks!

Reply:
Addressed in 31b8545ca42f0f629743bc1bade0a4c4b60c4027.